packages/rollup/install.md

```diff
@@ -122,5 +124,33 @@ You could use this value to perform extra production-only optimizations.
 For example you could define a constant for enabling Debug:
 
 ```javascript
-const DEBUG = process.env['COMPILATION_MODE'] === 'dbg';
+const DEBUG = process.env['COMPILATION_MODE'] === 'dbg'
```

Comment:
removing a semicolon?

Reply:
yeah, would you like me to put it back? just boiling things down to minimum

Reply:
I figure a full line of code would normally include it 🤷